prototool.yaml

```diff
@@ -0,0 +1,50 @@
+# Paths to exclude when searching for Protobuf files.
+#excludes:
+#  - path/to/a
+#  - path/to/b/file.proto
+
+# Protoc directives.
+protoc:
+  # The Protobuf version to use from https://github.com/protocolbuffers/protobuf/releases.
+  # You probably want to set this to make your builds completely reproducible.
+  version: 3.6.1
+
+  # Additional paths to include with -I to protoc.
+  # By default, the directory of the config file is included,
+  # or the current directory if there is no config file.
+  includes:
+    - core/api
```

Comment:
YAML formatting nit: lines starting with dash don't need to be indented further than the parent block

```
includes:
- core/api
```